.github/workflows/release_macOS.yml

```diff
@@ -86,3 +95,64 @@ jobs:
         run: yarn release:mac:arm64
         env:
           NOTARIZE: true
+      - name: Copy dist/latest-mac.yml to dist/latest-mac-arm64.yml
+        run: |
+          cp dist/latest-mac.yml dist/latest-mac-arm64.yml
+        shell: bash
+      - name: Save latest-mac-arm64.yml as an artifact
+        uses: actions/upload-artifact@v2
```

Comment:
there is a v4 version of this action https://github.com/actions/upload-artifact?tab=readme-ov-file#v4---whats-new